extensions/wikidata/src/org/openrefine/wikidata/qa/WikidataConstraintFetcher.java

```diff
@@ -340,4 +333,40 @@ public boolean usableOnItems(PropertyIdValue pid) {
         }
         return results;
     }
+
+    /**
+     * Returns the list of PropertyIdValues of conflicting statements
+     *
+     * @param pid:
+     *            the property having conflicts-with constraint
+     * @return
+     */
+    public List<PropertyIdValue> getConflictsWithProperties(PropertyIdValue pid) {
+        List<SnakGroup> specs = getSingleConstraint(pid, CONFLICTS_WITH_CONSTRAINT_QID);
```

Comment:
Because you are using `getSingleConstraint` here, this means that for properties with multiple conflicts-with constraints, you will only check the first one.

Reply:
Okay, let me try the alternatives. Thanks for the review. :)